.github/workflows/develop_commit_notification.yml

```diff
@@ -6,10 +6,12 @@ on:
 
 jobs:
   send_notifications:
-    name: Repository Dispatch
-    uses: peter-evans/repository-dispatch@v2
-    with:
-      token: ${{ secrets.REPO_ACCESS_TOKEN }}
-      repository: oppia/release-scripts
-      event-type: develop-commit
-      client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}"}'
+    if: github.repository == 'oppia/oppia'
```

Comment:
Please specify the OS this action will run on.

```yaml
runs-on: ubuntu-22.04
```

This is to ensure when GitHub actions changes the default OS this action does not fail.

Reply:
Done.